frontend/src/app/main/ui/ds/utilities/swatch.cljs

```diff
@@ -80,7 +84,9 @@
         button-type (if (not read-only?) "button" nil)
         size (or size "small")
         active (or active false)
-        gradient (:gradient background)
+        gradient-type (-> background :gradient :type)
+        gradient-stops (-> background :gradient :stops)
+        gradient-data (uc/gradient->css (gradient-data gradient-type gradient-stops))
```

Comment:
You don't need to define a function that simply builds a map, just build map inline without adding the function